client-core/src/lib.rs

```diff
@@ -36,3 +37,6 @@ pub use crate::transaction_builder::TransactionBuilder;
 pub use crate::unspent_transactions::{SelectedUnspentTransactions, UnspentTransactions};
 #[doc(inline)]
 pub use crate::wallet::{MultiSigWalletClient, WalletClient};
+
+#[macro_use]
+extern crate lazy_static;
```

Comment:
is this needed in the Rust 2018 edition?

Reply:
because #[maro_use], it's necessary
extern crate itself is no longer necessary.